var/spack/repos/builtin/packages/thrust/package.py

```diff
@@ -45,7 +52,35 @@ class Thrust(Package):
     version("1.8.3", sha256="2254200512fde7f4fd0fc74306286e192ea6ac9037576dbd31309c0579229dbb")
     version("1.8.2", sha256="83bc9e7b769daa04324c986eeaf48fcb53c2dda26bcc77cb3c07f4b1c359feb8")
 
-    def install(self, spec, prefix):
+    # leave the build system of pre-1.17 as originally implemented,
+    # but use CMake for newer versions
+    build_system(
+        conditional("cmake", when="@1.17:"), conditional("generic", when="@:1.16"), default="cmake"
+    )
+
+    # minimum CMake version is 3.15 for plain package installation (no
+    # examples, tests, etc)
+    depends_on("cmake@3.15:", when="@1.17:", type="build")
+
+
+# CMake provides an 'install' target, which provides the canonical
+# installation for Thrust (and CUB)
```

Comment:
Can we remove this:
```suggestion
```
? It's a redundant comment:

Reply:
done